test/functional/wallet_multiwallet.py

```diff
@@ -212,6 +230,18 @@ def wallet_file(name):
         w2 = node.get_wallet_rpc(wallet_names[1])
         w2.getwalletinfo()
 
+        if not self.options.usecli:
```

Comment:
Any reason to not test the cli? IIRC the reported bug was about the cli

Reply:
I just need to get the cli equivalent to `get_rpc_proxy`.

Reply:
Fixed.